py4DSTEM/process/diffraction/crystal.py

```diff
@@ -72,7 +72,7 @@ def __init__(
             ]
         )
 
-    def from_pymatgen_structure(structure, conventional_standard_structure=True):
+    def from_pymatgen_structure(structure=None, formula=None, space_grp=None, MaPKey= None, conventional_standard_structure=True):
```

Comment:
Please add information in the docstring about what the new options do, and how to properly format queries.

Reply:
Great...I will add info in the docstring